src/hypofuzz/hy.py

```diff
@@ -346,7 +346,8 @@ def _run_test_on(
         return data.as_result()
 
     def _report(self, report: Report) -> None:
-        (db := get_db()).save_metadata(self.database_key, report)
```

Comment:
I'm a fan of the walrus operator for "define in if guard, use in body", but I find this type of usage (define on first use) a bit gratuitous and unclear, idk. maybe I just need to get used to it 🙂

Reply:
I agree, this is less clear that I want the code to be.